packages/spaces/commands/vpn/update.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+const cli = require('heroku-cli-util')
+const co = require('co')
+const parsers = require('../../lib/parsers')()
+
+function check (val, message) {
+  if (!val) throw new Error(`${message}.\nUSAGE: heroku spaces:vpn:update --name office --cidrs 172.16.0.0/16,10.0.0.0/24 --space example-space`)
+}
+
+function * run (context, heroku) {
+  const lib = require('../../lib/vpn-connections')(heroku)
+
+  const space = context.flags.space
+  check(space, 'Space name required')
+
+  const name = context.flags.name || context.args.name
+  check(name, 'VPN name required')
+
+  let cidrs = context.flags.cidrs
+  check(cidrs, 'CIDRs required')
+  cidrs = parsers.splitCsv(cidrs)
+
+  yield cli.action(`Updating VPN Connection in space ${cli.color.green(space)}`, lib.patchVPNConnections(space, name, cidrs))
+}
```

Comment:
once we remove `co` above, we could change this to `async/await`:

```suggestion
async function run (context, heroku) {
  const lib = require('../../lib/vpn-connections')(heroku)

  const space = context.flags.space
  check(space, 'Space name required')

  const name = context.flags.name || context.args.name
  check(name, 'VPN name required')

  let cidrs = context.flags.cidrs
  check(cidrs, 'CIDRs required')
  cidrs = parsers.splitCsv(cidrs)

  await cli.action(`Updating VPN Connection in space ${cli.color.green(space)}`, lib.patchVPNConnections(space, name, cidrs))
}
```